client/i18n/locales/english/translations.json

```
@@ -483,6 +483,83 @@
     "executive": "Executive Director, freeCodeCamp.org",
     "verify": "Verify this certification at {{certURL}}",
     "issued": "Issued",
-    "fulltext": "<0>This certifies that</0> <1>{{user}}</1> <2>has successfully completed the freeCodeCamp.org</2> <3>{{title}}</3> <4>Developer Certification, representing approximately {{time}} hours of coursework.</4>"
+    "fulltext": "<0>This certifies that</0> <1>{{user}}</1> <2>has successfully completed the freeCodeCamp.org</2> <3>{{title}}</3> <4>Developer Certification, representing approximately {{time}} hours of coursework.</4>",
+    "project": {
+      "heading-legacy-full-stack": "As part of this Legacy Full Stack certification, {{user}} completed the following certifications:",
+      "heading": "As part of this certification, {{user}} built the following projects and got all automated test suites to pass:",
+      "solution": "solution",
+      "source": "source",
+      "footnote": "If you suspect that any of these projects violate the <0>academic honesty policy</0>, please <1>report this to our team</1>.",
```

```suggestion
      "footnote": "If you suspect that any of these projects violate the <1>academic honesty policy</1>, please <3>report this to our team.</3>",
```
The tag number generation is a bit odd, but these work for me locally.

---

These were working for me before as well when I tested it locally but sure I'll make this change :)

---

@ojeytonwilliams These were working for me before as well when I tested it locally, should I make this change?